spec/models/task_spec.rb

```diff
@@ -1,5 +1,43 @@
 require 'rails_helper'
 
 RSpec.describe Task, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  describe 'validation' do
+    it 'is valid with all attributes' do
+      user = FactoryBot.create(:user)
+      task = user.tasks.new(title: "aaa", content: "aaa", status: :todo)
+      expect(task).to be_valid
+      expect(task.errors).to be_empty
+    end
+
+    it 'is invalid without title' do
+      user = FactoryBot.create(:user)
+      task = user.tasks.new(title: "", content: "aaa", status: :todo)
+      expect(task).to be_invalid
+      expect(task.errors[:title]).to eq ["can't be blank"]
+    end
+
+    it 'is invalid without status' do
+      user = FactoryBot.create(:user)
+      task = user.tasks.new(title: "aaa", content: "aaa", status: nil)
+      expect(task).to be_invalid
+      expect(task.errors[:status]).to eq ["can't be blank"]
+    end
+
+    it 'is invalid with a duplicate title' do
+      user = FactoryBot.create(:user)
+      user.tasks.new(title: "aaa", content: "aaa", status: :todo).save
+      task_with_duplicated_title = user.tasks.new(title: "aaa", content: "aaa", status: :todo)
```

Comment:
`FactoryBot.create`を代入したローカル変数からtitleを取得して、重複データを作ってみましょう！
```suggestion
      task = FactoryBot.create(task:, user: user)
      task_with_duplicated_title = FactoryBot.build(:task, title: task.title, user: user)
```

Reply:
85d0f4c